### PR TITLE
Add zerg mercs

### PR DESCRIPTION
**Pull request by itsjustbones**
Related data pr https://github.com/Ziktofel/Archipelago-SC2-data/pull/269

## What is this fixing or adding?
Adds devouring ones and torrasque units

## How was this tested?
generated games and send upgrades to make sure it unlocked as expected

## If this makes graphical changes, please attach screenshots.
data PR has model changes, not this one

### CODE REVIEW

File: worlds/sc2/items.py [hunk 1]
```diff
@@ -1526,7 +1526,9 @@ def get_full_item_list():
     item_names.INFESTED_MEDICS: ItemData(600 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mercenary, 0, SC2Race.ZERG, origin={"ext"}),
     item_names.INFESTED_SIEGE_TANKS: ItemData(601 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mercenary, 1, SC2Race.ZERG, origin={"ext"}, classification=ItemClassification.progression),
     item_names.INFESTED_BANSHEES: ItemData(602 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mercenary, 2, SC2Race.ZERG, origin={"ext"}),
-    item_names.HUNTER_KILLERS: ItemData(603 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mercenary, 3, SC2Race.ZERG, origin={"ext"}),
+    item_names.DEVOURING_ONES_MERCS: ItemData(603 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mercenary, 3, SC2Race.ZERG, origin={"ext"}, classification=ItemClassification.progression),
```
Comment (Ziktofel) [inline worlds/sc2/items.py]: Not a logic item. Set the constant  name to `DEVOURING_ONES` to be consistent with the others

File: worlds/sc2/item_descriptions.py [hunk 2]
```diff
@@ -738,8 +740,8 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
...
     item_names.ROACH_PRIMAL_IGNITER_ASPECT: "Assault unit. Has an area-damage attack. Regenerates life quickly when burrowed. Can attack ground units. Morphed by merging two Roaches.",
     item_names.NYDUS_WORM: "Long-range transport network. Nydus Worms and Nydus Networks can load friendly ground units to be unloaded to any other Nydus structure on the map.",
-    item_names.OMEGA_WORM: "Long-range deployable base. Unable to load and unload units, but can generate Creep and Creep Tumors. Can also serve as a dropoff point for resources and can create Drones.",
-    item_names.ULTRALISK_TYRANNOZOR_ASPECT: "Heavy assault beast. Has a ground-area attack, and powerful anti-air attack. Morphed by merging two Ultralisks.",
+    item_names.OMEGA_WORM: "Long-range deployable base. Can't load and unload units, but can generate creep and Creep Tumors. Can also serve as a dropoff point for resources, like a Hatchery.",
+    item_names.ULTRALISK_TYRANNOZOR_ASPECT: "Heavy assault beast. Has a ground-area attack, and powerful anti-air attack.  Morphed by merging two Ultralisks.",
```
Comment (Ziktofel) [inline worlds/sc2/item_descriptions.py]: Double space

**Review by Ziktofel — CHANGES_REQUESTED** [review]

File: worlds/sc2/items.py (hunk 1 above)
Comment (itsjustbones) [reply worlds/sc2/items.py]: changed

File: worlds/sc2/item_descriptions.py (hunk 2 above)
Comment (itsjustbones) [reply worlds/sc2/item_descriptions.py]: Hmm, I've reverted the two changes here, not sure where it came from, probably something I reverted by accident.

**Pull request merged**